README.md

```diff
@@ -32,7 +35,7 @@ corresponding mapId form mapbox.
 If `divId` is not specified it defaults to 'map'.
 
 You can specify the zoom level, center of the map, and trigger an `onclick` event as attributes.
-The `onclick` event will return an instance of the event containing the lat,lng, among other properties.
+The `onclick` event will return an instance of the event containing the lat, lng, among other properties.
```

Comment:
Non-blocking but....
The `onclick` event will return an instance of the event containing the lat and lng, among other properties.
or
The `onclick` event will return an instance of the event containing the lat, lng and other properties.